packages/quantic/force-app/main/default/lwc/quanticRecentQueriesList/quanticRecentQueriesList.css

```diff
@@ -2,23 +2,17 @@
   display: flex;
   flex-direction: row;
   min-width: 0;
-  color: var(--brand-text-link, #0176d3);
 }
 
 .query-text__container {
   min-width: 0;
 }
 
 .recent-query__icon {
-  margin-right: .5rem;
-  margin-left: .25rem;
+  margin-right: 0.5rem;
+  margin-left: 0.25rem;
 }
 
 .empty-list-message {
-  color: var(--sds-c-color-gray-8, #8E959D);
-}
-
-li:hover {
-  text-decoration: underline;
-  cursor: pointer;
+  color: var(--sds-c-color-gray-8, #8e959d);
```

Comment:
This `--sds-c-color-gray-8` design token actually [isn't available](https://www.lightningdesignsystem.com/design-tokens/#category-color) to devs so apparently it'll always use the backup.

Reply:
ohh! i"ll change it to just the color code then